### ISSUE TITLE
Robolectric fallback in ModelInfo no longer valid (at least on Eclipse)

### ISSUE THREAD

**Issue by kevc** [pardom-zz/ActiveAndroid#80]
In scanForModel() in ModelInfo has a fallback for when Robolectric is being run. Unfortunately, the sourceDir in Eclipse no longer null when running Robolectric within Eclipse. The source directory ends up being the workspace directory that you are running within. 

As a result, when testing with Robolectric ActiveAndroid does not use the Robolectric fallback and tries to instantiate a new DexFile.

Is there another way to check that Robolectric is what is running in order for the Robolectric fallback to be executed?

**Comment by pardom-zz** [comment]
Duplicate of #69?

**Comment by kevc** [comment]
I don't believe so. I just ran the same test after build AA from the most recent code and got the same result.
